core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -96,8 +96,10 @@ use crate::transaction::LedgerTransaction;
 enum RocksDBColumnFamily {
     /// Committed transactions
     TxnByStateVersion,
-    TxnReceiptByStateVersion,
     TxnAccumulatorHashByStateVersion,
+    /// Receipts of committed transactions
+    LedgerReceiptByStateVersion,
+    LocalTransactionExecutionByStateVersion,
```

Comment:
_(note from author: this split at DB level is today strictly "conceptual", but @LukasGasior1 recommends it hard, and it may have real uses in future)_